src/redux/Books/BooksRedux.js

```diff
@@ -28,3 +35,20 @@ export const deleteBook = (id) => ({
   type: DELETEBOOK,
   payload: id,
 });
+
+const updateState = (list) => ({
+  type: UPDATESTATE,
+  payload: list,
+});
+
+export const getBooksfromAPI = () => (dispatch) => {
+  fetch(`${baseUrl}`, {
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json())
+    .then((json) => dispatch(updateState(json)));
+};
+ */
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.